app/pricing.py

```diff
@@ -62,7 +63,7 @@ def lookup_monthly_price(self, region_name: str, instance_type: str, operating_s
 
                 self.cache[cache_key] = hourly * HOURS_IN_A_MONTH
             return self.cache[cache_key]
-        except:
+        except:  # what to do about this?
```

Comment:
Looks like we have a straggler! Forgot to delete?

Reply:
Oops, yep, fixing now!